ruy/thread_pool.cc

```diff
@@ -123,13 +123,15 @@ class Thread {
   void ThreadFuncImpl() {
     ChangeState(State::Ready);
 
+    bool need_cpu_update = need_cpu_mask_update_;
```

Comment:
Can you explain why this local variable is required?
Looks like `need_cpu_mask_update_` is already a variable in each thread.